cmd/access/node_builder/staked_access_node_builder.go

```diff
@@ -264,6 +274,52 @@ func (builder *StakedAccessNodeBuilder) Build() (cmd.Node, error) {
 			// order for it to properly start and shut down, we should still return it as its own engine here, so it can
 			// be handled by the scaffold.
 			return builder.RequestEng, nil
+		}).
+		Component("execution data service", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+			err := os.MkdirAll(builder.executionDataDir, 0700)
+			if err != nil {
+				return nil, err
+			}
+
+			// TODO: we need to close this after the execution data services are stopped. it should probably be run as part of the node's postShutdown handler
+			datastore, err := badger.NewDatastore(builder.executionDataDir, &badger.DefaultOptions)
```

Comment:
Do we want to add TTL here?

Reply:
Not yet. We're only imported sealed blocks right now, so everything we fetch should be kept. We can add that in once we optimize to pre-fetching executed blocks

Reply:
But, once the Execution Data has been forwarded to DPS, we no longer need it on the Access Node, right?

Why don't we just add a command line option like what I did for execution node, and set the default to 0?

Reply:
Yea, I can add it as a command line arg. The main downside is it makes it easier for AN operators to not contribute to seeding the EDs for the network.

Reply:
We decided against using a time based expirations. For now, the start block is configurable, so an operator could just delete the db, and restart the node with a higher height. That's obviously not ideal, but should be good enough for v1 until we can add better tooling.